react/Payment.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useRef, useEffect } from 'react'
+
+if (window && window.document) {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  var postRobot = require('post-robot')
```

Comment:
Does
```suggestion
  const postRobot = await import('post-robot')
```
work in this case?

Reply:
it doesn't because the variable will be constrained to the scope of the `if` condition